components/psst/browser/content/psst_scripts_handler.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PSST_BROWSER_CONTENT_PSST_SCRIPTS_HANDLER_H_
+#define BRAVE_COMPONENTS_PSST_BROWSER_CONTENT_PSST_SCRIPTS_HANDLER_H_
+
+#include <memory>
+
+#include "base/component_export.h"
+
+class PrefService;
+
+namespace content {
+class WebContents;
+}  // namespace content
+
+namespace psst {
+
+class COMPONENT_EXPORT(PSST_BROWSER_CONTENT) PsstScriptsHandler {
```

Comment:
I don't think this needs to be exported? Also you can drop the cc file and just make this a pure virtual class.